RNTester/Podfile.lock

```diff
@@ -19,235 +19,235 @@ PODS:
-    - React-RCTNetwork (= 0.61.4)
-  - React-RCTImage (0.61.4):
-    - React-Core/RCTImageHeaders (= 0.61.4)
-    - React-RCTNetwork (= 0.61.4)
-  - React-RCTLinking (0.61.4):
-    - React-Core/RCTLinkingHeaders (= 0.61.4)
-  - React-RCTNetwork (0.61.4):
-    - React-Core/RCTNetworkHeaders (= 0.61.4)
-  - React-RCTPushNotification (0.61.4):
-    - React-Core/RCTPushNotificationHeaders (= 0.61.4)
-  - React-RCTSettings (0.61.4):
-    - React-Core/RCTSettingsHeaders (= 0.61.4)
-  - React-RCTTest (0.61.4):
-    - React-Core (= 0.61.4)
-  - React-RCTText (0.61.4):
-    - React-Core/RCTTextHeaders (= 0.61.4)
-  - React-RCTVibration (0.61.4):
-    - React-Core/RCTVibrationHeaders (= 0.61.4)
-  - ReactCommon/jscallinvoker (0.61.4):
+    - React-cxxreact (= 0.61.13)
+    - React-jsi (= 0.61.13)
+  - React-jsinspector (0.61.13)
+  - React-RCTActionSheet (0.61.13):
+    - React-Core/RCTActionSheetHeaders (= 0.61.13)
+  - React-RCTAnimation (0.61.13):
+    - React-Core/RCTAnimationHeaders (= 0.61.13)
+  - React-RCTBlob (0.61.13):
+    - React-Core/RCTBlobHeaders (= 0.61.13)
+    - React-Core/RCTWebSocket (= 0.61.13)
+    - React-jsi (= 0.61.13)
+    - React-RCTNetwork (= 0.61.13)
+  - React-RCTImage (0.61.13):
+    - React-Core/RCTImageHeaders (= 0.61.13)
+    - React-RCTNetwork (= 0.61.13)
+  - React-RCTLinking (0.61.13):
+    - React-Core/RCTLinkingHeaders (= 0.61.13)
+  - React-RCTNetwork (0.61.13):
+    - React-Core/RCTNetworkHeaders (= 0.61.13)
+  - React-RCTPushNotification (0.61.13):
+    - React-Core/RCTPushNotificationHeaders (= 0.61.13)
+  - React-RCTSettings (0.61.13):
+    - React-Core/RCTSettingsHeaders (= 0.61.13)
+  - React-RCTTest (0.61.13):
+    - React-Core (= 0.61.13)
+  - React-RCTText (0.61.13):
+    - React-Core/RCTTextHeaders (= 0.61.13)
+  - React-RCTVibration (0.61.13):
+    - React-Core/RCTVibrationHeaders (= 0.61.13)
+  - ReactCommon/jscallinvoker (0.61.13):
     - DoubleConversion
     - Folly (= 2018.10.22.00)
     - glog
-    - React-cxxreact (= 0.61.4)
-  - ReactCommon/turbomodule/core (0.61.4):
+    - React-cxxreact (= 0.61.13)
+  - ReactCommon/turbomodule/core (0.61.13):
     - DoubleConversion
     - Folly (= 2018.10.22.00)
     - glog
-    - React-Core (= 0.61.4)
-    - React-cxxreact (= 0.61.4)
-    - React-jsi (= 0.61.4)
-    - ReactCommon/jscallinvoker (= 0.61.4)
-  - ReactCommon/turbomodule/samples (0.61.4):
+    - React-Core (= 0.61.13)
+    - React-cxxreact (= 0.61.13)
+    - React-jsi (= 0.61.13)
+    - ReactCommon/jscallinvoker (= 0.61.13)
+  - ReactCommon/turbomodule/samples (0.61.13):
     - DoubleConversion
     - Folly (= 2018.10.22.00)
     - glog
-    - React-Core (= 0.61.4)
-    - React-cxxreact (= 0.61.4)
-    - React-jsi (= 0.61.4)
-    - ReactCommon/jscallinvoker (= 0.61.4)
-    - ReactCommon/turbomodule/core (= 0.61.4)
+    - React-Core (= 0.61.13)
+    - React-cxxreact (= 0.61.13)
+    - React-jsi (= 0.61.13)
+    - ReactCommon/jscallinvoker (= 0.61.13)
+    - ReactCommon/turbomodule/core (= 0.61.13)
```

Comment:
Hmm, now noticing that these are no longer overriden to be 1000, probably since the version naming change? Will take a look.

Reply:
https://github.com/microsoft/react-native-macos/pull/349